pkg/metrics/metrics.go

```diff
@@ -87,57 +85,3 @@ func (m *Metric) Write(s *strings.Builder) {
 	writeFloat(s, m.Value)
 	s.WriteByte('\n')
 }
-
-func labelsToString(m *strings.Builder, keys, values []string) {
```

Comment:
Why not keep those here as they are only used by `func (m *Metric) Write(s *strings.Builder) {`?

Reply:
I think initially I moved some other functions from `pkg/collectors` to `metrics/utils.go` but didn't need them at the end, so moved these as well, can move back 👍